api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -53,6 +53,13 @@ message RedisProxy {
     // need to be known to the cluster manager. If the command cannot be redirected, then the
     // original error is passed downstream unchanged. By default, this support is not enabled.
     bool enable_redirection = 3;
+
+    // Maximum size of buffer before flush is triggered. If this is unset, the buffer flushes
+    // whenever it receives data and performs no batching.
+    uint32 max_buffer_size_before_flush = 4;
+
+    // Buffer is flushed every N milliseconds (unless the buffer is filled before the timer fires)
```

Comment:
Can you expand? What buffer? How does it relate to `max_buffer_size_before_flush`? Also what is the behavior when this is unset?

Reply:
Added comments